### PR TITLE
feat(shorebird_cli): `shorebird init` detects iOS flavors

### PR DESCRIPTION
**Pull request by felangel**
## Description

- feat(shorebird_cli): `shorebird init` detects iOS flavors (part of #910)
  - flavors are detected without breaking the `shorebird.yaml` format

## Type of Change

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: packages/shorebird_cli/lib/src/commands/init_command.dart [hunk 1]
```diff
@@ -71,10 +73,21 @@ If you want to reinitialize Shorebird, please run "shorebird init --force".''');
...
     try {
-      productFlavors = await gradlew.productFlavors(Directory.current.path);
+      final flavors = await Future.wait([
+        _maybeGetAndroidFlavors(Directory.current.path),
+        _maybeGetiOSFlavors(Directory.current.path),
+      ]);
+      androidFlavors = flavors[0];
+      iosFlavors = flavors[1];
+      productFlavors = <String>{
+        if (androidFlavors != null) ...androidFlavors,
```
Comment (eseidel) [inline packages/shorebird_cli/lib/src/commands/init_command.dart:88]: Does ...?androidFlavors work?

Comment (eseidel) [inline packages/shorebird_cli/lib/src/commands/init_command.dart:88]: Does ...?androidFlavors work?

**Review by bryanoltman — APPROVED** [review]
Nice!

File: packages/shorebird_cli/lib/src/commands/init_command.dart (hunk 1 above)
Comment (felangel) [reply packages/shorebird_cli/lib/src/commands/init_command.dart:88]: Yup they should be unaffected

Comment (eseidel) [reply packages/shorebird_cli/lib/src/commands/init_command.dart:88]: Oh, I was just asking if the null-aware splat operator was a thing, I think it is and would save you a few characters typing.

**Pull request merged**